bin/release-rewards.js

```diff
@@ -7,25 +7,46 @@ import * as SparkImpactEvaluator from '@filecoin-station/spark-impact-evaluator'
 import readline from 'node:readline/promises'
 import pRetry from 'p-retry'
 import beeper from 'beeper'
+import pMap from 'p-map'
 
 process.title = 'release-rewards'
 const { RPC_URL = 'https://api.node.glif.io/rpc/v1', WALLET_SEED } = process.env
 
+const provider = new ethers.JsonRpcProvider(RPC_URL)
```

Comment:
I think we should configure Glif authorisation header and abort the script if no Glif API key is not provided because we are going to make a lot of RPC API requests in the `pMap` call below.

Reply:
https://github.com/filecoin-station/spark-rewards/pull/46/commits/7e6dfb421ca388c372d418f0880dbb3e970faa0b